go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.13
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.0
 	github.com/ElrondNetwork/elrond-go-logger v1.0.7
-	github.com/ElrondNetwork/elrond-vm-common v1.3.17-0.20220914092453-aec8d55cd00c
+	github.com/ElrondNetwork/elrond-vm-common v1.3.17-0.20220919101133-149c0edcc821
```

Comment:
`elrond-vm-common` now has `v1.3.17`, consider referencing it here.

Reply:
this is with async calls - which has a separate vm-common for now.